packit_service/worker/handlers/abstract.py

```diff
@@ -209,6 +211,49 @@ def get_packit_commands_from_comment(
     return []
 
 
+class CeleryTask:
+    def __init__(self, task: Task):
+        self.task = task
+
+    @property
+    def retries(self):
+        return self.task.request.retries
+
+    def is_last_try(self) -> bool:
+        """
+        Returns True if the current celery task is run for the last try.
+        More info about retries can be found here:
+        https://celeryproject.readthedocs.io/en/latest/userguide/tasks.html#retrying
+        """
+        return self.retries == self.get_retry_limit()
```

Comment:
I'd update this to `self.retries >= self.get_retry_limit()` just to be a bit safer.